test/functional/apps/console/_autocomplete.ts

```diff
@@ -217,6 +217,7 @@ GET _search
         for (const keyPress of keyPresses) {
           await PageObjects.console.sleepForDebouncePeriod();
           log.debug('Key', keyPress);
+          // @ts-ignore
```

Comment:
I synced with @yuliacech, adding a ts-ignore for now as this test breaks the PRs which are introducing changes to the FTs suites. They are going to fix it properly on another PR.